### PR TITLE
[OSDOCS-9157]:What's New Announcement for new force-delete cluster flag "--best-effort"

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.14+
Issue:

https://issues.redhat.com/browse/OSDOCS-9157
Link to docs preview:

https://69657--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_release_notes/rosa-release-notes#rosa-q4-2023_rosa-whats-new
QE review:
- [x] QE has approved this change.

Reviewers: This "What's new with ROSA" doc addition is based on the new feature described and reviewed here: https://issues.redhat.com/browse/OSDOCS-9038.   
Additional information:

image

### CODE REVIEW

File: rosa_release_notes/rosa-release-notes.adoc [hunk 1]
```diff
@@ -17,6 +17,9 @@ toc::[]
 === Q4 2023
 
 * **ROSA CLI update.** The ROSA CLI (`rosa`) was updated to a new version. For information about what has changed in this release, see the link:https://github.com/openshift/rosa/releases/tag/v1.2.32[ROSA CLI release notes]. For more information about the ROSA CLI (`rosa`), see xref:../cli_reference/rosa_cli/rosa-get-started-cli.adoc#rosa-about_rosa-getting-started-cli[About the ROSA CLI].
+* **Delete cluster command enhancement.** With the release of ROSA CLI (`rosa`) version 1.2.31, a new argument, `--best-effort`, was added that allows you to force-delete clusters when using the `rosa delete cluster` command. For more information, see xref:../cli_reference/rosa_cli/rosa-manage-objects-cli.adoc#rosa-delete-cluster_rosa-managing-objects-cli[delete cluster].
```
Comment (mburke5678) [inline rosa_release_notes/rosa-release-notes.adoc:20]:
```suggestion
* **Delete cluster command enhancement.** With the release of ROSA CLI (`rosa`) version 1.2.31, the `--best-effort` argument was added, which allows you to force-delete clusters when using the `rosa delete cluster` command. For more information, see xref:../cli_reference/rosa_cli/rosa-manage-objects-cli.adoc#rosa-delete-cluster_rosa-managing-objects-cli[delete cluster].
```

**Comment by mletalie** [comment]
> @mletalie Not sure why the _Delete cluster_ entry is listed twice. A nit or two. Otherwise LGTM

Thanks for catching that, @mburke5678. I know I was having some merge/rebase issues last night, so I am sure something got messed up. I was looking right at it and didn't see it! :(  Appreciate it.

**Pull request merged**